CactbotOverlay/FFXIVProcess.cs

```diff
@@ -368,6 +371,63 @@ public enum EntityJob {
       return active_process_id == process_.Id;
     }
 
+    public class StatusEffect {
+      private static Dictionary<int, float> maxDurations = new Dictionary<int, float>();
```

Comment:
I think it'd be better to have this as a table in Javascript rather than sending this on every effect every frame.

Reply:
This isn't sent with the JSEvent at all, it is only used to internally store the maximum durations by id for already seen status effects. Each status effect gets only its own max duration in the constructor.